app/library.py

```diff
@@ -26,8 +25,7 @@ def tts_to_mp3(text: str) -> bytes:
     mp3_convert.save("speech.mp3")
     with open("app/speech.mp3", "rb") as f:
         final = f.read()
-        encoded = base64.b64encode(final)

```

Comment:
We need this line. Otherwise, the request handler will return an error because `bytes` is not JSON serializable. It needs to be base64 encoded as a string first.

The return type on line 21 should be `str`, however.

Reply:
Ignore this mishap. My environment was not updated to the latest changes made by Thomas. I'll revert this.